mule-bpm-dummy-flowable-activity/src/main/java/org/flowable/mule/MuleSendActivityBehavior.java

```diff
@@ -48,27 +41,5 @@ public void execute(DelegateExecution execution) {
         throw new RejectedExecutionException("Dummy implementation cannot execute activities");
     }
 
-    public void setEndpointUrl(Expression endpointUrl) {
```

Comment:
Ei poisteta näitä settereitä, Flowable käyttää niitä. Voidaan poistaa propertyt ja tyhjentää metodit mutta siitä saattaa tulla uusi haju.